### ISSUE TITLE
Source code

### ISSUE THREAD

**Issue by romain-neil** [nspeed-app/nspeed#3]
As far as I know, the license used in this project (CC-BY-NC, https://creativecommons.org/licenses/by-nc/4.0/), we should be able to adapt/remix and/or transform.

And, for an "open source" project, it's common to provide the source code ...

Sincerely,

**Comment by kgersen** [comment]
Yes sorry if it's not clear , the licence is not yet definitive. This all repo for now is just a placeholder.

The source code will be released with v1.0 as stated in the README.md
